src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -22,24 +22,24 @@
 
 package network.brightspots.rcv;
 
-import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.logging.Level;
 
 @SuppressWarnings("WeakerAccess")
 public class Main extends GuiApplication {
 
+  public static String APP_NAME = "RCVRC Universal Tabulator";
+  public static String APP_VERSION = "1.0.0";
+
   // function: main
   // purpose: main entry point to the rcv tabulator program
   // param: args command line argument array
   // returns: N/A
   public static void main(String[] args) {
-    try {
-      Logger.setup();
-    } catch (IOException exception) {
-      System.err.print(String.format("Failed to start system logging!\n%s", exception.toString()));
-    }
+    System.out.println(String.format("Launching %s version %s", APP_NAME, APP_VERSION));
```

Comment:
```suggestion
    System.out.println(String.format("%s version %s", APP_NAME, APP_VERSION));
```
Just for the sake of having at least a little less duplication than the line that's officially logged.